services/core-web/src/components/Forms/EditMinespaceUser.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Field, reduxForm } from "redux-form";
+import { Form } from "@ant-design/compatible";
+import "@ant-design/compatible/assets/index.css";
+import { Button, Col, Row } from "antd";
+import { required, requiredList } from "@common/utils/Validate";
+import { resetForm } from "@common/utils/helpers";
+import RenderField from "@/components/common/RenderField";
+import * as FORM from "@/constants/forms";
+import { renderConfig } from "@/components/common/config";
+import CustomPropTypes from "@/customPropTypes";
+
+const propTypes = {
+  mines: CustomPropTypes.options.isRequired,
+  // eslint-disable-next-line react/no-unused-prop-types
+  minespaceUserEmailHash: PropTypes.objectOf(PropTypes.any).isRequired,
+  handleSubmit: PropTypes.func.isRequired,
+  handleChange: PropTypes.func.isRequired,
+  handleSearch: PropTypes.func.isRequired,
+};
+
+const minespaceUserNotExists = (value, allValues, props) =>
+  value && !(value in props.minespaceUserEmailHash)
+    ? undefined
+    : "A user with this email already exists";
+
```

Comment:
This validation isn't needed in the Edit Modal as the username cannot be edited. Please remove.